deploy/README.md

```diff
@@ -0,0 +1,128 @@
+<!--
+Copyright 2022 The DAPHNE Consortium
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# DAPHNE Deployment
+
+### Where to Start
+
+Use [deploy/deploy-distributed-on-slurm.sh](./deploy-distributed-on-slurm.sh) to start. When executed without parameters, it prints out the help message.
+
+### Overview
+
+This directory [deploy/](../deploy/) can be used to **deploy DAPHNE** Runtime Systems.
+Deploying allows the [source code](../src/) to be:
+- used for building of the Daphne System using [build.sh](../build.sh),
+- packaged,
+- delivered and installed on to the Deployment Platform (e.g. HPC) as a unifying Deployment Platform of these Runtime Systems, and 
+- run on the Resources of a set of distributed components.
+It can also be used to just try out DAPHNE on a single machine.
+
+Once a deployment is running, the Distributed Workers (running `DistributedWorker` processes) are running in their Runtime Systems and the main Target Runtimes use them to execute Daphne scripts in a Distributed Deployment.
+
+### Deployment Scheme
+
+DAPHNE Deployment Scheme encompasses the following:
+
+- Deployment Platform (e.g. an HPC with SLURM support)
+  - Runtime System(s)
+    - Runtime System for the Main Target
+      - `daphne` Target
+    - Runtime System for the Distributed Worker Target
+      - `DistributedWorker` Target
+
```

Comment:
Thanks. This is now addressed in the commit e8932a4126548d32739d3f7965abe9153cd358b4.
Instead of the file artifacts coming from of the `build.sh` (Target type in `build.sh`), this Deployment Scheme paragraph now rather refers to the machine types (Compilation node, Login node, Compute Node(s)).
The file artifacts (targets) are removed. The "coordinator" is now put in font `coordinator`, to refer to a special case of a component in the Daphne system and linked to `daphne`.